<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>yggdrasil_editor</class>
 <widget class="QMainWindow" name="yggdrasil_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yggdrasil Editor</string>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>567</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="fileNew"/>
    <addaction name="fileOpen"/>
    <addaction name="fileSave"/>
    <addaction name="fileSaveAs"/>
    <addaction name="separator"/>
    <addaction name="fileExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="editCut"/>
    <addaction name="editCopy"/>
    <addaction name="editPaste"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuDocks">
     <property name="title">
      <string>Dock Windows</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuToolbars">
     <property name="title">
      <string>Toolbars</string>
     </property>
    </widget>
    <addaction name="menuDocks"/>
    <addaction name="menuToolbars"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="menuToolBar">
   <property name="windowTitle">
    <string>Menu</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="fileToolBar">
   <property name="windowTitle">
    <string>File</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="fileNew"/>
   <addaction name="fileOpen"/>
   <addaction name="fileSave"/>
  </widget>
  <widget class="QToolBar" name="editToolBar">
   <property name="windowTitle">
    <string>Edit</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="editCut"/>
   <addaction name="editCopy"/>
   <addaction name="editPaste"/>
  </widget>
  <widget class="QDockWidget" name="dockScreen">
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>148</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Screens</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockScreenContents">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>9</number>
     </property>
     <item>
      <widget class="editor::custom::FilterWidget" name="screenTreeFilter"/>
     </item>
     <item>
      <widget class="editor::ScreenTree" name="screenTree"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockObject">
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>148</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Objects</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockObjectContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="editor::custom::FilterWidget" name="objectTreeFilter"/>
     </item>
     <item>
      <widget class="editor::ObjectTree" name="objectTree"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockProperties">
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockPropertiesContents"/>
  </widget>
  <widget class="QToolBar" name="typeToolBar">
   <property name="windowTitle">
    <string>View type</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockLog">
   <property name="windowTitle">
    <string>Messages</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="fileNew">
   <property name="icon">
    <iconset resource="resources/application.qrc">
     <normaloff>:/filenew.xpm</normaloff>:/filenew.xpm</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="fileOpen">
   <property name="icon">
    <iconset resource="resources/application.qrc">
     <normaloff>:/fileopen.xpm</normaloff>:/fileopen.xpm</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="fileSave">
   <property name="icon">
    <iconset resource="resources/application.qrc">
     <normaloff>:/filesave.xpm</normaloff>:/filesave.xpm</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="fileSaveAs">
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
  </action>
  <action name="editCut">
   <property name="icon">
    <iconset resource="resources/application.qrc">
     <normaloff>:/editcut.xpm</normaloff>:/editcut.xpm</iconset>
   </property>
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
  </action>
  <action name="editCopy">
   <property name="icon">
    <iconset resource="resources/application.qrc">
     <normaloff>:/editcopy.xpm</normaloff>:/editcopy.xpm</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="editPaste">
   <property name="icon">
    <iconset resource="resources/application.qrc">
     <normaloff>:/editpaste.xpm</normaloff>:/editpaste.xpm</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="fileExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="lockToolBars">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lock toolbars</string>
   </property>
   <property name="toolTip">
    <string>Lock the toolbars</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>editor::custom::FilterWidget</class>
   <extends>QLineEdit</extends>
   <header>editor/filterwidget.h</header>
  </customwidget>
  <customwidget>
   <class>editor::ScreenTree</class>
   <extends>QTreeView</extends>
   <header>editor/screentree.h</header>
  </customwidget>
  <customwidget>
   <class>editor::ObjectTree</class>
   <extends>QTreeView</extends>
   <header>editor/objecttree.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>fileExit</sender>
   <signal>triggered()</signal>
   <receiver>yggdrasil_editor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
